TYPO3.Fluid/Classes/TYPO3/Fluid/Core/Parser/SyntaxTree/ObjectAccessorNode.php

```diff
@@ -88,10 +88,22 @@ public static function getPropertyPath($subject, $propertyPath, RenderingContext
     {
         $propertyPathSegments = explode('.', $propertyPath);
         foreach ($propertyPathSegments as $pathSegment) {
+            $isHasDirectAccess = false;
             try {
-                $subject = ObjectAccess::getProperty($subject, $pathSegment);
+                if (preg_match('/^(is|has)([A-Z].*)/', $pathSegment, $matches) > 0) {
+                    $subject = ObjectAccess::getProperty($subject, lcfirst($matches[2]));
```

Comment:
And checking again, I realize that you kept the duplication for greater B/C..
But maybe we can still keep the readability by doing sth like:

```
if (ObjectAccess::isPropertyGettable($$subject, lcfirst($matches[2]))) {
  $propertyName = lcfirst($matches[2]);
}
```

then instead